### ISSUE TITLE
CTCAlignmentsSearch: masking the alignment logits for optimisation

### ISSUE THREAD

**Issue by dijksterhuis** [dijksterhuis/cleverspeech-exp#1]
Need to insert a [mask variable](https://github.com/dijksterhuis/cleverspeech-exp/blob/master/CTCAlignments/custom_defs.py#L75).
